Sources/TensorFlow/Operators/LinearAlgebra.swift

```diff
@@ -177,6 +177,25 @@ public func trace<T: TensorFlowNumeric>(_ matrix: Tensor<T>) -> Tensor<T> {
     return matrix.diagonalPart().sum(squeezingAxes: -1)
 }
 
+/// Computes the determinant of an optionally batched matrix.
+/// 
+/// - Parameter matrix: A tensor of shape `[..., M, M]`.
+/// - Returns: A tensor containing the determinants of all input submatrices.
+@inlinable 
+func det<T: TensorFlowFloatingPoint>(_ matrix: Tensor<T>) -> Tensor<T> {
```

Comment:
```suggestion
func determinant<T: TensorFlowFloatingPoint>(_ matrix: Tensor<T>) -> Tensor<T> {
```

Reply:
Obsolete. Let's continue to use `det` after discussion in https://github.com/tensorflow/swift-apis/pull/604#issuecomment-573653157.